apps/sms/views/conversation/style/message.css

```diff
@@ -142,6 +142,20 @@
   color: #d1eeee;
 }
 
+.message.incoming .message-details .late-arrival-notice {
```

Comment:
@julienw I would let `.message.incoming .late-arrival-notice` just for the sake of semantic because the late arrival notice only applies to incoming messages.